qdrant-landing/content/documentation/embeddings/nomic.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Nomic"
+weight: 1100
+---
+
+# Nomic
+
+The `nomic-embed-text-v1` model is an [open source](https://github.com/nomic-ai/contrastors) 8192 context length text encoder.
+It is available on [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1), but the easiest way to use it is 
+through the [Nomic Text Embeddings](https://docs.nomic.ai/reference/endpoints/nomic-embed-text), either by using the official
+Python client or by sending HTTP requests directly.
```

Comment:
Unclear: what's the "official" Python client? (I'm guessing we should drop the word)

```suggestion
While you can find it on the [Hugging Face Hub](https://huggingface.co/nomic-ai/nomic-embed-text-v1), 
you may have more success through the [Nomic Text Embeddings](https://docs.nomic.ai/reference/endpoints/nomic-embed-text).
Once installed, you can configure it with the official Python client or through direct HTTP requests.
```

Reply:
Added a suggestion above on wording to make it more concise.